packages/office-ui-fabric-react/src/utilities/selection/SelectionZone.tsx

```diff
@@ -34,6 +34,7 @@ const SELECTION_DISABLED_ATTRIBUTE_NAME = 'data-selection-disabled';
 const SELECTION_INDEX_ATTRIBUTE_NAME = 'data-selection-index';
 const SELECTION_TOGGLE_ATTRIBUTE_NAME = 'data-selection-toggle';
 const SELECTION_INVOKE_ATTRIBUTE_NAME = 'data-selection-invoke';
+const SELECTION_INVOKE_TOUCH_ATTRIBUTE_NAME = 'data-selection-touch-invoke';
```

Comment:
is there a test that could be added to make sure this keeps working

Reply:
Have added a couple of unit tests for both the scenario where touch targets are disabled, and when they are enabled :)